### ISSUE TITLE
Real world concerns and weaponizing

### ISSUE THREAD

**Issue by wc-zach** [dxa4481/Pastejacking#11]
So I totally understand how this exploit works, but I'm curious who we think the target audience is and how well could one actually weaponize such an exploit. To me it seems like a pretty big stretch to get a high value target to copy and paste code off of an untrusted website and into a shell, but I'm open to other opinions and ideas.

First off, this exploit is easily defeated by pasting your clipboard into a benign text editor such as Sublime Text or Notepad. Nothing new here, but if this exploit does start to become a serious problem then I would assume this easy procedure would protect people 100% of the time.

But more interesting to me is the intended targets. It sounds to me like students, web hobbyists, perhaps a rushed dev-ops engineer would be the most likely phishing target. I just can't think of that many users who use the terminal and aren't power users. It's not like MS Word or Photoshop is gonna execute the hidden commands if a user pastes this message into them. But maybe I'm being naive, as we are all becoming more and more computer proficient and maybe more people are using the terminal.

I'm curious to hear any responses or thoughts about this, and thanks again for making the effort to publish this exploit and bring more attention to it.

**Comment by crazy4groovy** [comment]
I don't think (I could be wrong) that anyone is going to post exploit situations or ideas here. Your imagination is the limit.

The valid point is that JavaScript can manipulate a system function (Ctrl+C) that the user reasonably expects to work a certain way.

**Comment by wc-zach** [comment]
To be clear, I'm not looking for bash script examples for injecting an ssh user or anything, I can write that myself. I'm asking for high level ideas or thoughts on why this would be a practical attack vector.

The project's point is valid until the browser vendors fix, in a technical sense, but it's not a very practical exploit compared to click jacking or xsrf.

Also, using the edit menu or right clicking both don't mutate the clipboard in the example. I was able to get a bit more coverage by using the `copy` event instead of a `keydown` event, but it still doesn't work across all browsers. 

The easiest way I have found to defeat this exploit is to select copy from the edit menu with your mouse. In my limited tests (OS X only), it seems this method fails to mutate the clipboard everytime
